go/common/sgx/pcs/policy.go

```diff
@@ -0,0 +1,83 @@
+package pcs
+
+import (
+	"fmt"
+)
+
+// QuotePolicy is the quote validity policy.
+type QuotePolicy struct {
+	// Disabled specifies whether PCS quotes are disabled and will always be rejected.
+	Disabled bool `json:"disabled,omitempty" yaml:"disabled,omitempty"`
+
+	// TCBValidityPeriod is the validity (in days) of the TCB collateral.
+	TCBValidityPeriod uint16 `json:"tcb_validity_period" yaml:"tcb_validity_period"`
+
+	// MinTCBEvaluationDataNumber is the minimum TCB evaluation data number that is considered to be
+	// valid. TCB bundles containing smaller values will be invalid.
+	MinTCBEvaluationDataNumber uint32 `json:"min_tcb_evaluation_data_number" yaml:"min_tcb_evaluation_data_number"`
+
+	// FMSPCBlacklist is a list of hexadecimal encoded FMSPCs specifying which processor
+	// packages and platform instances are blocked.
+	FMSPCBlacklist []string `json:"fmspc_blacklist,omitempty" yaml:"fmspc_blacklist,omitempty"`
+
+	// TDX is an optional TDX-specific policy. In case this is nil, TDX quotes are disallowed.
+	TDX *TdxQuotePolicy `json:"tdx,omitempty" yaml:"tdx,omitempty"`
+}
+
+// TdxQuotePolicy is the TDX-specific quote policy.
+type TdxQuotePolicy struct {
+	// AllowedTdxModules are the allowed TDX modules. Empty to allow ANY Intel-signed module.
```

Comment:
Currently only Intel-signed modules exist. Why is this scary? Note that this is an additional restriction as all TDX modules are already verified against the TCB infos.

Reply:
I thought this was scary because if a user wanted to restrict modules to only one and forgot to add it to the list, he would mistakenly allow all Intel-signed modules. Which is not what you would think first when you see an empty access/white list.

Reply:
The idea is that an empty policy would allow all Intel-signed modules. I guess we could explicitly have an `Option<Vec<_>>`? Or do you have any other suggestions.

Reply:
Keep it as it is. Since we allow only Intel signed modules and not all, maybe it is not that scary.